website/src/main/tut/docs/users/configuration.md

```diff
@@ -109,4 +111,38 @@ debug.printSymbols = true
 
 To build custom rules see {% doc_ref Rule Authors %}.
 
+## Per source file
+
+It's possible to supress false positives in the source code itself to have a more fine grain on what to report and what not to report. There is two techniques you could use to disable linting: disable/enable until end of file or disable per expression.
+
+### Enable or Disable until the EOF
+
+```scala
+
+// scalafix:off
+foo(null)
+1.asInstanceOf[String]
+// scalafix:on
 
+```
+
+### Disable for an expression
+
+```scala
+List(1, "") // scalafix:ok
+```
+
+### Selectively disabling rules
+
+Both technique above can selectively disable a list of rules. You can provide the list of rules to be disabled, separated by `,`.
+
+```scala
+// scalafix:off Disable.null, Disable.asInstanceOf
+foo(null)
+1.asInstanceOf[String]
+// scalafix:on
+```
+
+```scala
+List(1, "") // scalafix:ok NoInfer.any
+```
```

Comment:
Should we also document that scalafix will report warning for unused `off/on/ok` comments? I think it's a pretty great selling point